cfme/tests/v2v/test_csv_import.py

```diff
@@ -269,17 +253,36 @@ def test_csv_archived_vm(appliance, infra_map, archived_vm):
     """Test csv with archived vm name
     Polarion:
         assignee: ytale
-        caseimportance: high
         caseposneg: positive
-        testtype: functional
         startsin: 5.10
         casecomponent: V2V
         customerscenario: true
         initialEstimate: 1/8h
-        subcomponent: RHV
-        upstream: yes
     """
     content = "Name\n{}".format(archived_vm)
     error_msg = "VM is inactive"
     assert import_and_check(appliance, infra_map, error_msg,
                             content=content, table_hover=True)
+
+
+@pytest.mark.uncollectif(lambda provider: provider.one_of(RHEVMProvider))
+def test_csv_security_group_flavor(appliance, infra_map, valid_vm, provider):
+    """Test csv with openstack security group and flavor
+    Polarion:
+        assignee: ytale
+        caseposneg: positive
+        startsin: 5.10
+        casecomponent: V2V
+        customerscenario: true
+        initialEstimate: 1/4h
+    """
+    security_group = provider.data.security_groups.admin[1]
+    flavor = provider.data.flavors[1]
```

Comment:
These required provider data fields should be included in the provider marker for the test function.

Otherwise we'll end up picking providers from yaml that don't include these fields, and the test will fail with KeyError/AttributeError/IndexError here.

At a minimum, wrap these statements in `try:except:pytest.skip` pattern, but ideally use `ProviderFilter.required_fields` to limit provider collection in the first place.

Reply:
Make sense, let me update it.

Reply:
Updated with try:except:pytest.skip